PracticeProgram.scala

```diff
@@ -0,0 +1,15 @@
+package pioneer.practice
+
+import Practice.square
+import Practice.repeat
+
+object PracticeProgram extends App {
+  print("Enter a number: ")
+  val inputNumber = scala.io.StdIn.readInt()
+  val result = square(inputNumber)
+  println(s"The square of ${inputNumber} is ${result}.")
+  print("Enter a String: ")
+  val inputString = scala.io.StdIn.readLine()
+  val string = repeat(inputString)
+  println("repeating: " + string)
+}
```

Comment:
It looks like this file was in the top directory of the repository. It's redundant here (there's already a version of this file in `/src/main/scala/pioneer/practice`.